tethys_gizmos/static/tethys_gizmos/js/tethys_map_view.js

```diff
@@ -572,6 +580,8 @@ var TETHYS_MAP_VIEW = (function() {
           layer.tethys_legend_classes = current_layer.legend_classes;
           layer.tethys_legend_extent = current_layer.legend_extent;
           layer.tethys_legend_extent_projection = current_layer.legend_extent_projection;
+          layer.tethys_editable = current_layer.editable;
```

Comment:
I wanted things to be consistent with the other parameters previously built *shrug*